Pre_processing.py

```
@@ -15,37 +15,31 @@
     #Contains various functions to prepare the data for modelling
-    def load_data(self,filepath):
-         #Low the data from the filepath
-        df_out = spark.read.csv(filepath,header = False)
-        return df_out
 
-    def append_headers(self,df):
-        #Give the dataframe some column headers
-        column_names = ["target", "id", "date", "flag", "user", "text"]
-        df_out = df.toDF(*column_names)
+    def load_data(self,data_location):
+        df_out = spark.table(data_location)
```

Ruff

Undefined name `spark`

(F821)